src/utils/errors/types.ts

```diff
@@ -0,0 +1,14 @@
+import { HttpStatusCode } from "axios";
+import { includes, isObject, isString, values } from "lodash/fp";
+
+export type httpError = {
+  message: string;
+  code: HttpStatusCode;
+};
+
+export const isHttpError = (possibleError: unknown): possibleError is httpError =>
```

Comment:
למה בקובץ types? 
אפשר אולי להשתמש בפונ loadash בשביל להיפטר מכל ה&&?

Reply:
the types file seemed like the appropriate place to put it. isHttpError is a type predicate. and why lodash?